src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.dto.request.UserRegistrationDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private AuthenticationService authenticationService;
+
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public void register(@RequestBody @Validated UserRegistrationDto userRegistrationDto) {
```

Comment:
```suggestion
    public void register(@RequestBody @Valid UserRegistrationDto userRegistrationDto) {
```
https://www.baeldung.com/spring-valid-vs-validated